src/main/java/net/sargue/mailgun/MailBuilder.java

```diff
@@ -225,6 +225,26 @@ public MailBuilder content(Body body) {
         return text(body.text()).html(body.html());
     }
 
+    /**
+     * Sets the name of the pre saved template that will be used to be sent by Mailgun.
+     *
+     * @param templateName the name of pre saved template on Mailgun
+     * @return this builder
+     */
+    public MailBuilder template(String templateName){
```

Comment:
Please keep existing formatting convetions: space between ) and { at the end of the line.